src/web-ui/src/public/Live.vue

```diff
@@ -208,7 +208,7 @@ export default {
     },
     async loadPlayer () {
       const mediaPlayerScript = document.createElement("script");
-      mediaPlayerScript.src = "https://player.live-video.net/1.0.0/amazon-ivs-player.min.js";
+      mediaPlayerScript.src = "https://player.live-video.net/1.18.0/amazon-ivs-player.min.js";
```

Comment:
1.19.0 was released on May 23rd, any reason to not use it ?
Ref: https://docs.aws.amazon.com/ivs/latest/userguide/release-notes.html

Reply:
Regarding updating the player. It just means having to update and retest, which I would rather do as part of our next review of all dependencies.

The API GW routes were decided based on what was required by the current frontend.  Service to service communication is direct, as before, so there are no restrictions.  If we wanted to expose sensitive functions to the frontend, then we should update the authorization Lambda to ensure that the user has sufficient privileges - something we don't do now, but should be easy to add now that we have the authorizer lambda in the APIGW.